package.json

```diff
@@ -13,27 +13,29 @@
     "url": "https://github.com/plotly/react-plotly.js-editor/issues"
   },
   "scripts": {
-    "make:combined-translation-keys":
-      "babel-node utils/findTranslationKeys.js && babel-node utils/combineTranslationKeys.js",
+    "lint": "prettier --write \"src/**/*.js\"",
+    "make:combined-translation-keys": "babel-node utils/findTranslationKeys.js && babel-node utils/combineTranslationKeys.js",
     "make:translation-keys": "babel-node utils/findTranslationKeys.js",
-    "make:lib":
-      "mkdirp lib && babel src --out-dir lib --ignore=__tests__/* --source-maps && npm run make:lib:css",
-    "make:lib:css":
-      "mkdirp lib && node-sass src/styles/main.scss > lib/react-plotly.js-editor.css",
-    "make:dist":
-      "mkdirp dist && browserify src/PlotlyEditor.js -o ./dist/PlotlyEditor.js -t [ babelify --presets [ es2015 react ] ] -t browserify-global-shim --standalone createPlotlyComponent && uglifyjs ./dist/PlotlyEditor.js --compress --mangle --output ./dist/PlotlyEditor.min.js --source-map filename=dist/PlotlyEditor.min.js.map && make:dist:css",
-    "make:dist:css":
-      "mkdirp dist && node-sass --output-style compressed src/styles/main.scss > dist/react-plotly.js-editor.css",
+    "make:lib": "mkdirp lib && babel src --out-dir lib --ignore=__tests__/* --source-maps && npm run make:lib:css",
```

Comment:
@VeraZab I think you and @aulneau have squabbling .json formatting. I don't think we should use prettier on anything outside `/src` or at least not on anything other than *.js files.

Reply:
Yeah, sometimes my muscle memory causes me to autoformat anything after I hit save. But it's strange, my current package.json is formatted correctly:

<img width="1211" alt="screen shot 2018-01-03 at 12 03 03 pm" src="https://user-images.githubusercontent.com/11803153/34533057-16fcc9e8-f07e-11e7-86ed-308b82c1ba36.png">